### ISSUE TITLE
No compatible devices could be found! // 60HE+ // MacOS Sonoma 14.5

### ISSUE THREAD

**Issue by DAB513** [WootingKb/wooting-analog-midi#43]
Does not detect plugged in wooting 60HE+
MacOS Sonoma 14.5

wooting-analog-sdk is installed correctly
For apple silicon M2 Pro I had to:
sudo cp -r /opt/homebrew/Cellar/wooting-analog-sdk/0.7.0/lib /usr/local/

<img width="796" alt="image" src="https://github.com/user-attachments/assets/601c596a-eada-4990-8809-7361499e7d04">

**Comment by simon-wh** [comment]
You need v0.7.5 of the Analog SDK for the 60HE+ to be detected https://github.com/WootingKb/wooting-analog-sdk/releases/tag/v0.7.5

**Comment by DAB513** [comment]
I downloaded this release but I am unsure how to properly update the sdk due to unmatched file names from brew install

**Comment by simon-wh** [comment]
What file names are unmatched? The files themselves should be the same naming, but the internal structure of the files is not the same as what you need on your system

**Comment by DAB513** [comment]
Sorry about the confusion. Do you have a proper way to install this release on apple silicon Mac?

**Comment by simon-wh** [comment]
No worries, the install for Mac currently is a bit jank. I had tried to update our homebrew tap with the latest version, but I remember running into some issues.

https://github.com/wootingkb/wooting-analog-sdk?tab=readme-ov-file#manual-1

There are some manual steps here, I noticed the plugins bit is a bit out of date. For the plugin bit, you just need to copy
`wrapper/sdk/libwooting_analog_plugin.dylib` to `/usr/local/share/WootingAnalogPlugins/wooting-analog-plugin/`
